packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -809,7 +809,7 @@ describe('MenuTrigger', function () {
     });
   });
 
-  it('closes if menu is tabbed away from', async function () {
+  it('does not close if menu is tabbed away from', async function () {
```

Comment:
ah, yep, there it is
I think I'm ok with it, how often do we really think this is happening